docs/src/modules/components/ChartComponentsGrid.js

```diff
@@ -81,7 +81,7 @@ export default function ChartComponentsGrid() {
               })}
             />
             <Typography
-              component="h2"
+              component="h3"
```

Comment:
Feel like better represent the content of the page https://blog.pope.tech/2022/08/04/5-heading-accessibility-issues-and-how-to-fix-them/.